synapse/lib/storm.lark

```diff
@@ -178,7 +185,16 @@ embedquery: _EMBEDQUERYSTART query "}"
 _EMBEDQUERYSTART: "${"
 
 // An unquoted storm string constant.  Must start with a word char, ?, or - and not contain certain punctutation
-NONQUOTEWORD: /[\w\-?][^ \t\n),=\]}|]*/
+NONQUOTEWORD: /[\w\-\+\?\*][^ \t\n),=\]}|]*/
+
+// An unquoted string within a whitespace list
+WORDTOKN: /[\w\+\-\?\*\/\\][^ \t\r\n\=\|\}\)]*/
+
+// An unquoted string within a list syntax
+LISTTOKN: /[\w\+\-\?\*\/\\]([^,)]*[^\s,)])*/
```

Comment:
Let's add equals to the list of disallowed character in a list token (for future-proofing kwargs).

Reply:
All of these tokens should not be able to be started with $.#{. That is probably a pre-existing bug.